<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="initButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>190</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>init</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bindButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>190</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>bind</string>
   </property>
  </widget>
  <widget class="QPushButton" name="freeButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>450</y>
     <width>190</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>free</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
